src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -830,6 +837,7 @@ def setUp(self):
         get_job_manager(self, class_name_default=None)
 
         # Mark the end of setup
+        self.log_step('setUp(): Setup complete')
```

Comment:
I would say this isn't even a step at all. Just informational/debugging

Reply:
I agree, but to set get the elapsed time for the agent startup and reset it for the next test step this is easier than requiring a log_step call in each test_* method.

Reply:
Right. But we could just as easily add a direct start/stop time for various actions instead of using `log_step`. For example, in this `setUp` function just directly save a start time and end time, rather than indirectly relying on `log_step`